README.md

```diff
@@ -1,5 +1,21 @@
 # T-asapp
 Proyecto ingeniería software
 
-PROYECTO DE GESTIÓN LOGÍSTICA DE ENVÍO DE PAQUETERÍA
+PROYECTO RECETAS PERSONALIZADAS
+
+Este proyecto pertenece a la materia de Ingeniería del Software del Máster en Programación y Diseño de Aplicaciones
+de EDIAE y T-Systems.
+
+Somos jóvenes estudiantes/trabajadores a los que nos gustaría tener una mejor alimentación adaptada a nuestras necesidades individuales, teniendo en cuenta desde intolerancias y alergias alimentarias hasta objetivos nutricionales específicos. El problema del que partimos es la gran inversión de tiempo que necesitamos para encontrar recetas que suplan esta necesidad. Resulta muy frustrante y agotador tener que estar pensando qué comer, sin tener que consultar múltiples fuentes hasta dar con una receta que cumpla con todos nuestros requisitos.
+
+El tiempo que se invierte en esta búsqueda junto con la falta de datos nutricionales en las recetas hace que sea una pérdida completa de tiempo sin los resultados deseados. 
+Cada persona puede tener unas prioridades a la hora de enfocar su alimentación, unos objetivos, limitaciones, preferencias. 
+
+En este sentido, también podrían tenerse en cuenta recetas que incluyan exclusivamente productos de temporada o proximidad, puesto que también habrá usuarios que prioricen este tema por su impacto medioambiental.
```

Comment:
"También" no es algo que se suela incluir en la descripción de un problema. Si queréis describir diferentes problemas, hacedlo de esa forma; eso se convertirán en diferentes "personas" en la fase siguiente. Es simplemente un cambio semántico, pero tiene cierta importancia.